src/styles/blocks/card.scss

```diff
@@ -0,0 +1,93 @@
+.card {
+  box-sizing: border-box;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  background: #fff;
+  padding: 16px;
+  padding-top: 32px;
+  width: 200px;
+
+  &__photo {
+    display: block;
+    width: 160px;
+    height: 134px;
+    margin: 0 auto 40px;
+  }
+
+  &__title {
+    margin: 0 0 4px;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+
+  }
+
+  &__code {
+    margin: 0 0 16px;
+    color: #616070;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+
+  }
+
+  &__rating {
+    display: flex;
+    justify-content: space-between;
+  }
+
+  &__reviews {
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    line-height: 18px;
+    margin: 0 0 16px;
+  }
+
+  &__label {
+    color: #616070;
+    font-size: 12px;
+    font-weight: 400;
+  }
+
+  &__value {
+    font-size: 16px;
+    font-weight: 700;
+  }
+
+  &__buy-button {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    padding: 12px 70px;
+    border: 1px solid #f3f3f3;
+    border-radius: 5px;
+    background-color: #00acdc;
+    cursor: pointer;
+
+    a {
+      text-decoration: none;
+      text-transform: uppercase;
+      color: #fff;
+      font-size: 14px;
+      font-weight: 700;
+      line-height: 16px;
+      padding: 0;
+      border: none;
+    }
```

Comment:
Use styles for classes not for main selectors 
Don't use tag <button>  try to visite QnA and solve your problem with tests.